### PR TITLE
Finally solved issue #1734

### PR DESCRIPTION
**Pull request by quadcopterman**

### CODE REVIEW

**Comment by Kaz-ookid** [comment]
my bad for marking it as ready to review, forgot some tests are to come

**Comment by quadcopterman** [comment]
This PR is another way to solve issue #1734 (much much simpler than my previous attempt) : just let the day comparison go to -1 when comparing the start date of an event (in case midnight is just between start date and now). If this event is too far in the past, it will be handeled by computeTimeInSeconds() as Instant also contains the date.
A test was added to make sure that a 1 day old date won't work and that this issue won't occur again

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/event/AbstractEventCreationFragment.kt [hunk 1]
```diff
@@ -202,7 +202,9 @@ abstract class AbstractEventCreationFragment : Fragment() {
     startDateEditText?.setText("")
     startDate = null
 
-    if (compareWithNowByDay(newDate) < 0) {
+    // let the comparison go to -1 for cases where the time is just after midnight
+    // this is handled just fine by computeTimeInSeconds() as an Instant also contains the date
+    if (compareWithNowByDay(newDate) < -1) {
```
Comment (matteosz) [inline fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/event/AbstractEventCreationFragment.kt:207]: Refactor opportunity: this check appears twice (here and line 224), so it'd be better to extract it into a variable

**Review by matteosz — COMMENTED** [review]
LGTM! 

Just 1 small comment

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/event/AbstractEventCreationFragment.kt (hunk 1 above)
Comment (quadcopterman) [reply fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/lao/event/AbstractEventCreationFragment.kt:207]: Yes, thanks for spotting that, I'll correct it

**Review by Kaz-ookid — APPROVED** [review]

**Review by matteosz — APPROVED** [review]
LGTM!

**Pull request merged**